js/flightlog_fields_presenter.js

```diff
@@ -1023,6 +1023,83 @@ function FlightLogFieldPresenter() {
             'debug[6]':'Not Used',
             'debug[7]':'Not Used',
         },
+        'FAILSAFE' : {
+            'debug[all]': 'Failsafe',
+            'debug[0]': 'Failsafe Phase switch',
+            'debug[1]': 'Failsafe State',
+            'debug[2]': 'Receiving data from Rx',
+            'debug[3]': 'Failsafe Phase',
+        },
+        'GYRO_CALIBRATION' : {
+            'debug[all]': 'Gyro Calibration',
+            'debug[0]': 'Gyro Calibration X',
+            'debug[1]': 'Gyro Calibration Y',
+            'debug[2]': 'Gyro Calibration Z',
+            'debug[3]': 'Calibration Cycles remaining',
+        },
+        'ANGLE_MODE' : {
+            'debug[all]': 'Angle Mode',
+            'debug[0]': 'Angle Target',
+            'debug[1]': 'Angle Error',
+            'debug[2]': 'Angle Feedforward',
+            'debug[3]': 'Angle Current',
+        },
+        'ANGLE_TARGET' : {
+            'debug[all]': 'Angle Target',
+            'debug[0]': 'Angle Target',
+            'debug[1]': 'Sin Angle',
+            'debug[2]': 'Current PID Setpoint',
+            'debug[3]': 'Angle Current',
+        },
+        'CURRENT_ANGLE' : {
+            'debug[all]': 'Current Angle',
+            'debug[0]': 'Current Angle X',
+            'debug[1]': 'Current Angle Y',
+            'debug[2]': 'Current Angle Z',
+        },
+        'DSHOT_TELEMETRY_COUNTS' : {
+            'debug[all]': 'DShot Telemetry Counts',
+            'debug[0]': 'DShot Telemetry Debug[0] + 1',
+            'debug[1]': 'DShot Telemetry Debug[1] + 1',
+            'debug[2]': 'DShot Telemetry Debug[2] + 1',
+            'debug[3]': 'Preamble Skip',
+        },
+        'RPM_LIMIT' : {
+            'debug[all]': 'RPM Limit',
+            'debug[0]': 'Average RPM',
+            'debug[1]': 'Average RPM (unsmoothed)',
+            'debug[2]': 'RPM Limit throttle scale',
+            'debug[3]': 'Throttle',
+            'debug[4]': 'Error',
+            'debug[5]': 'Proportional',
+            'debug[6]': 'Integral',
+            'debug[7]': 'Derivative',
+        },
+        'RC_STATS' : {
+            'debug[all]': 'RC Stats',
+            'debug[0]': 'Average Throttle',
+        },
+        'MAG_CALIB' : {
+            'debug[all]': 'Mag Calibration',
+            'debug[0]': 'Mag X',
+            'debug[1]': 'Mag Y',
+            'debug[2]': 'Mag Z',
+            'debug[3]': 'Norm / Length of magADC',
+            'debug[4]': 'Estimated Mag Bias X',
+            'debug[5]': 'Estimated Mag Bias Y',
+            'debug[6]': 'Estimated Mag Bias Z',
+            'debug[7]': 'Mag Bias Estimator',
```

Comment:
This can be called 'Lambda', it represents an estimate of the precision of the estimate of the bias values.